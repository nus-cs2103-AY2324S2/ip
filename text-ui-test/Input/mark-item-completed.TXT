todo test
deadline something /by A day
event lecture /from Today /to Next Year
list
mark 2
list
mark 1
list
mark 3
list
bye