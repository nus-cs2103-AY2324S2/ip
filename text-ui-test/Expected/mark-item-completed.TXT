Hello from
 ______     ______     __  __
/\  __ \   /\  __ \   /\ \/ /
\ \ \/\ \  \ \  __ \  \ \  _-.
 \ \_____\  \ \_\ \_\  \ \_\ \_\
  \/_____/   \/_/\/_/   \/_/\/_/

----------------------------------------------
Welcome! I'm Professor oak
What can I do for you?
----------------------------------------------
Added new Todo: test
----------------------------------------------
----------------------------------------------
Added new Deadline: something with Due Date: A day
----------------------------------------------
----------------------------------------------
Added new Event: lecture occurring from Today to Next Year
----------------------------------------------
----------------------------------------------
1. [T][ ] test
2. [D][ ] something (by: A day)
3. [E][ ] lecture (from: Today to: Next Year)
----------------------------------------------
----------------------------------------------
Ok! I've marked Task 2 as completed!
----------------------------------------------
----------------------------------------------
1. [T][ ] test
2. [D][X] something (by: A day)
3. [E][ ] lecture (from: Today to: Next Year)
----------------------------------------------
----------------------------------------------
Ok! I've marked Task 1 as completed!
----------------------------------------------
----------------------------------------------
1. [T][X] test
2. [D][X] something (by: A day)
3. [E][ ] lecture (from: Today to: Next Year)
----------------------------------------------
----------------------------------------------
Ok! I've marked Task 3 as completed!
----------------------------------------------
----------------------------------------------
1. [T][X] test
2. [D][X] something (by: A day)
3. [E][X] lecture (from: Today to: Next Year)
----------------------------------------------
----------------------------------------------
Goodbye! Hope to see you again!
----------------------------------------------
